server.go

```diff
@@ -62,6 +62,7 @@ func (s *server) Start() (bool, error) {
 	}, bufSize)
 
 	<-s.persister.ConfigChangedWatcher()
+	s.createBucketContainer()
```

Comment:
got it! yeah might be slightly obvious. we could also embed the method to `Start`..